lib/addons/flagger/index.ts

```diff
@@ -0,0 +1,66 @@
+import { HelmAddOn, HelmAddOnUserProps, HelmAddOnProps } from "../helm-addon";
+import { ClusterInfo } from "../../spi";
```

Comment:
please group import together. So whatever is coming from ../../spi should be a single import.

Reply:
Oh did not notice I did that.